src/dashboard-widgets/dashboard-widgets.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {css} from '@emotion/core'
+import Parcel from 'single-spa-react/parcel'
+
+const patientDashboardParcels = [
+  () => import('./basic-info/basic-info.parcel').then(m => m.default),
+]
+
```

Comment:
@joeldenning why are we using promises with the import statement?

Reply:
https://github.com/tc39/proposal-dynamic-import and https://webpack.js.org/guides/code-splitting/#dynamic-imports

Reply:
https://github.com/CanopyTax/single-spa-react#parcel-props